From 03d0937f69ff82c74da7afb286a037b523044671 Mon Sep 17 00:00:00 2001
From: Adriana Kobylak <anoo@us.ibm.com>
Date: Tue, 17 May 2016 11:48:59 -0500
Subject: [PATCH 51/56] dts/aspeed-barreleye: Add motherboard eeprom

The motherbaord eeprom in Barreleye is accessible through i2c4 address
0xA8 (54). Adding it to the device tree since its vpd contains the
system UUID which is needed to be added to the inventory.

Signed-off-by: Adriana Kobylak <anoo@us.ibm.com>
[reformatted commit message]
Signed-off-by: Joel Stanley <joel@jms.id.au>
---
 arch/arm/boot/dts/aspeed-bmc-opp-barreleye.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed-bmc-opp-barreleye.dts b/arch/arm/boot/dts/aspeed-bmc-opp-barreleye.dts
index a06252a..0e567e8 100644
--- a/arch/arm/boot/dts/aspeed-bmc-opp-barreleye.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-opp-barreleye.dts
@@ -82,6 +82,11 @@
 						reg = <0x10>;
 						sense-resistor = <500>;
 					};
+					eeprom@54 {
+						compatible = "atmel,24c256";
+						reg = <0x54>;
+						pagesize = <64>;
+					};
 				};
 				i2c5: i2c-bus@180 {
 					adm1278@10 {
-- 
1.9.1

